CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Added
+- Add demo for extracted audio tracks and subtitle settings from the settings panel direct into the controlBar
```

Comment:
When you add an entry into the `Added` section, please don't start the entry with `Add`, as it is totally redundant. For example ``Demo with separate `SettingsPanel`s for subtitles and audio tracks`` would be sufficient.